raiden/ui/cli.py

```diff
@@ -598,6 +598,9 @@ def run_app(
     privatekey_hex = hexlify(privatekey_bin)
     config['privatekey_hex'] = privatekey_hex
 
+    if not eth_rpc_endpoint.startswith('http://'):
```

Comment:
This will break in case e.g. `https` was provided. There's also other schemas/protocols we may want to support [soon] (i.e. `ipc`, `ws`). I'd suggest checking the `schema` with` urllib.parse.urlparse`, and preprending `http://` only if `None`.

Reply:
A loosely related issue here is this: https://github.com/raiden-network/raiden/issues/2096

Reply:
I was under the impression that geth only provided an http interface in addition to IPC. Anyway, will update my fix.

Reply:
`geth` also provides the websockets interface on port `8546` by default, and web3.py added support to it not long time ago: ethereum/web3.py#708
Anyway, we aren't doing that right now, just a headsup.